parameter_files/fates_params_default.cdl

```diff
@@ -15,45 +15,12 @@ dimensions:
 	fates_plant_organs = 4 ;
 	fates_string_length = 60 ;
 variables:
-	double fates_history_ageclass_bin_edges(fates_history_age_bins) ;
```

Comment:
@rgknox I'm seeing a lot of resorting of the names; should I fix this?

Reply:
Hi @glemieux do you want me to look at this now, or hold off until the variables are sorted back the previous way?

Reply:
@rosiealice I just fixed it, please go ahead and review.  Thanks.